zc_common/jwt_auth/tests.py

```diff
@@ -109,6 +112,47 @@ def setUp(self):
         # Create an instance of the Permission class in child class
         self.permission_obj = None
 
+    def test_revoked_permissions(self):
```

Comment:
This is doing way too much testing in just one test. It's very aggressive! Also very difficult to understand what is going on. If this fails for any reason, the dev trying to figure out what went wrong will have a hard time understanding it. This mixin is widely used in many repos and using it requires a lot of knowledge.